src/sinks/datadog/metrics/tests.rs

```diff
@@ -1,4 +1,4 @@
-#![allow(clippy::print_stdout)] // tests
+#![cfg(feature = "datadog-metrics-integration-tests")]
```

Comment:
Can you rename this file to `integration_tests.rs` and do the inclusion of it, including the `cfg` annotation, in mod.rs?

Additionally, the `cfg` annotation should also match for both `test` and `feature = "datadog-metrics-integration-tests"`.